spectrum/autism_prevalence_map/templates/autism_prevalence_map/base.html

```diff
@@ -20,84 +20,55 @@
     <meta property="og:image" content="https://prevalence.spectrumnews.org/static/autism_prevalence_map/img/PrevalenceMapSG.png" />
     <meta property="og:description" content="The map features a collection of studies on autism prevalence around the world. It highlights places where information is available — and places where information is missing." />
 
-    <!-- Mobile Specific Metas
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <!-- CSS libraries
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-    <!-- <link rel="stylesheet" href="https://use.typekit.net/hjy6pmo.css"> -->
-    <script src="https://use.typekit.net/gnz4dab.js"></script>
-    <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/vendor/css/bootstrap.min.css' %}">
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
-    <!-- Custom site-wide CSS
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-    <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/css/style.css' %}">
+    {% if environment == 'development' %}
+        <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/dist/main.css' %}">
+    {% else %}
+        <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/dist/main.min.css' %}">
+    {% endif %}
 
     {% block css_block %} {% endblock %}
 
-    <!-- Favicon
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-    <link rel="icon" type="image/png" href="{% static 'autism_prevalence_map/img/favicon.png' %}">
+    <link rel="icon" type="image/svg+xml" href="{% static 'autism_prevalence_map/img/favicon.svg' %}">
 
-    <!-- [Google Analytics/Optimize] -->
     <style>.async-hide { opacity: 0 !important} </style>
     <script async src="https://www.googletagmanager.com/gtag/js?id=G-BKVX7LHVRB"></script>
-    <script>
```

Comment:
I don't know what it is; I missed it because, in my first pass, I was really just trying to set up the logo, colors, and typography and not doing a full audit of the code

Reply:
OK maybe there should be another ticket for cleaning up, then, like removing files from the vendor folder and cleaning up base.html?